neat_internal.h

```diff
@@ -21,6 +21,7 @@
 #endif
 
 #include "neat_log.h"
+#include "neat_stat.h"
```

Comment:
> On 15 Jun 2016, at 11:17, Kristian A. Hiorth <notifications@github.com> wrote:
> 
> In neat_internal.h <https://github.com/NEAT-project/neat/pull/90#discussion_r67126561>:
> 
> > @@ -21,6 +21,7 @@
> >  #endif
> >  
> >  #include "neat_log.h"
> > +#include "neat_stat.h"
> Why include neat_stat.h here, there don't seem to be any further changes in this header requiring anything from there.
> 
This was an artefact of some testing using a struct in neat_stat.h for a list of all flows. I took that away, so there’s no need to keep this include, especially if we decide to use a different data structure for that functionality. 

I will take that line away.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/NEAT-project/neat/pull/90/files/24b0d23d9732e1813e296d2d221beb851bec481a#r67126561>, or mute the thread <https://github.com/notifications/unsubscribe/ACejY34Pca-yOyogFHKmSv4M_vBbnOvgks5qL8MkgaJpZM4Izozz>.
>